jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1172,19 +1176,16 @@ public synchronized void start() {
 	 *
 	 * Release the lock before calling threadgroup's remove method for this thread.
 	/*[ENDIF]*/
-
 	try {
-		synchronized(lock) {
+		synchronized (lock) {
 			if (started) {
 				/*[MSG "K0341", "Thread is already started"]*/
 				throw new IllegalThreadStateException(com.ibm.oti.util.Msg.getString("K0341")); //$NON-NLS-1$
 			}
 
 			/*[PR 115667, CMVC 94448] In 1.5, thread is added to ThreadGroup when started */
 			group.add(this);
```

Comment:
You haven't addressed the concerns noted in https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/387#discussion_r788155380.

Reply:
> The call to ThreadGroup.add() should not be within the try block;

Was this the concern?
Is there a scenario this can cause a problem?

Reply:
If an exception is thrown from `add()` then we should assume the thread has not been added to the group and so it would be inappropriate to remove it from the group.

Reply:
Moving `group.add(this)` out of the try block will introduce a time window that multiple `group.add(this)` can happen, and multiple duplicate thread instances are to be added into the `group`.
Removing a thread not already added into the `group` is a no-op, and probably in a slow path anyway since `group.add()` threw `IllegalThreadStateException` because `group.isDestroyed` is true.
I think current behaviour is safer.

Reply:
Moved `started` check and `group.add(this)` out of the `try` block.

Reply:
I don't think moving that code outside of the `synchronized (lock)` block is appropriate.

Reply:
Added `synchronized (lock)` for `started` & `group.add(this)`.